config/routes.rb

```diff
@@ -181,6 +181,8 @@
       end
 
       resources :sites, path: "locations", on: :member, except: %i[destroy show]
+
+      get "/allocations", on: :member, to: "providers/allocations#index"
```

Comment:
```suggestion
      scope module: "providers" do
        resources :allocations, only: [:index]
      end
```

and all the crazy thing it enlists...